velox/vector/LazyVector.h

```diff
@@ -135,6 +135,9 @@ class LazyVector : public BaseVector {
       }
       return;
     }
+    if (type_->kind() == TypeKind::ROW && !vector_) {
```

Comment:
This change seems unrelated. Is this fixing some bug? Would you submit it in a separate PR?